tests/repository_test.py

```diff
@@ -1128,3 +1128,27 @@ def test_non_installable_hook_error_for_additional_dependencies(store, caplog):
         'using language `system` which does not install an environment.  '
         'Perhaps you meant to use a specific language?'
     )
+
+
+def test_lua_hook(tempdir_factory, store):
+    _test_hook_repo(
+        tempdir_factory, store, 'lua_repo',
+        'hello-world-lua', [], b'hello world\n',
+    )
+
+
+def test_local_lua_additional_dependencies(store):
+    config = {
+        'repo': 'local',
+        'hooks': [{
+            'id': 'local-lua',
+            'name': 'local-lua',
+            'entry': 'lua',
+            'language': 'lua',
+            'args': ['-e', 'require "inspect"; print("hello world")'],
+            'additional_dependencies': ['inspect'],
```

Comment:
`inspect` pretty prints tables (because the default print of a table spits out a fairly useless memory address). This seemed like a tame additional dependency to check against.